src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/SwitchToNewOrgButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const linkStyle = {
+  color: 'black',
+  textDecoration: 'underline',
+  cursor: 'pointer',
+}
+
+export const SwitchToNewOrgButton = (url: string): JSX.Element => {
+  const handleSwitchToNewOrg = () => {
```

Comment:
Is this handler necessary? can this be done by putting an `href` on the link? Should the href go on the anchor and an `event.preventDefault()` go here for usability (if an anchor tag has an `href` and the anchor has focus, hitting `enter` will follow the link

Reply:
It sure isn't 🙃 . Removed it and converted it to a standard `href`.